x-pack/plugins/lens/public/editor_frame_service/editor_frame/suggestion_panel.scss

```diff
@@ -89,6 +89,12 @@
   flex-grow: 0;
 }
 
-.lnsSuggestionPanel__applyChangesPrompt {
+.lnsSuggestionPanel__applyChangesPrompt.euiPanel {
```

Comment:
Curious: why is added `.euiPanel` here?

Reply:
Had to create more CSS specificity to beat out the native EUI styling.

Reply:
Would it be better to use an `!important` on the styles that require the added specificity, rather than targeting an EUI class (as those are subject to change without our knowledge)?

Reply:
Sure. Makes sense to me.